src/configs/BUILD

```diff
@@ -282,152 +98,27 @@ xnnpack_cc_library(
 )
 
 xnnpack_cc_library(
-    name = "dwconv_config",
-    srcs = ["dwconv-config.c"],
-    deps = [
-        ":config_hdrs",
-        ":hardware_config",
-        "//:common",
-        "//:init_once",
-        "//:logging",
-        "//:microfnptr_h",
-        "//:microkernel_hdrs",
-        "//:microparams_init",
-    ] + xnnpack_select_if(
-        "//:cpuinfo_enabled",
-        ["@cpuinfo"],
-    ),
-)
-
-xnnpack_cc_library(
-    name = "gemm_config",
-    srcs = ["gemm-config.c"],
+    name = "microkernel_configs",
+    hdrs = [
+        "//:src/xnnpack/config.h",
+    ],
+    srcs = MICROKERNEL_CONFIG_SRCS,
```

Comment:
Is reduce-config.c and unary-elementwise-config.c getting compiled twice now?

Can src/pack-lh.c just depend on all the configs, rather than treating the reduce and unary elementwise configs as "special"?